src/Raven.Server/Exceptions/SmugglerException.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using Raven.Client.Exceptions;
+
+namespace Raven.Server.Exceptions;
+
+public class SmugglerException : RavenException
```

Comment:
Why do we add a new type of exception?

Reply:
What "additional data" you mean? There is no extra / new property added here?

Reply:
I see but everything is concatenated into single string - exception message. Is it worth to add a new exception type?

Also after deserializing such exception on the client side it will be propagated as `RavenException` anyway since the one you added is defined in `src/Raven.Server/Exceptions` so it's not known on Raven.Client side. If you want it to be properly deserialized as `SmugglerException` you'd need to define it in `Raven.Client`